pkgs/by-name/ni/nixos-render-docs/package.nix

```
@@ -38,9 +38,12 @@ python.pkgs.buildPythonApplication rec {
     src = ./src;
   };
 
-  nativeBuildInputs = with python.pkgs; [
-    setuptools
-    pytestCheckHook
+  nativeCheckInputs = [
+    python.pkgs.pytestCheckHook
+  ];
+
+  nativeBuildInputs = [
+    python.pkgs.setuptools
   ];
```

You could move `setuptools` in `build-system` as well, entirely eliminating the `nativeBuildInputs`.
```suggestion
  build-system = [ python.pkgs.setuptools ];
```

---

You're right! Thanks for the suggestion.